src/renderers/dom/client/syntheticEvents/SyntheticEvent.js

```diff
@@ -195,3 +198,44 @@ SyntheticEvent.augmentClass = function(Class, Interface) {
 PooledClass.addPoolingTo(SyntheticEvent, PooledClass.fourArgumentPooler);
 
 module.exports = SyntheticEvent;
+
+
+// Utility functions
```

Comment:
Nit: I don't think this comment is necessary. Do we use similar comments in the codebase?